Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<h3>1a. Assets HOT-1 and HOT-3 </h3><p> </p><p> </p><h4>HOT-1 <em>(Hands on Training Unit 1) </em>Description </h4><p> </p><p> </p><p><strong>[VERIFY SENTENCES WITH QUESTION MARKS ????]</strong> &nbsp;</p><p> </p><p>This unit is used to pump water and other materials <strong>????</strong> from one tank another tank.</p><p> </p><p>There are 3 tanks in the HOT-1 unit: </p><ul><li>Feed Tank T-101: 300-gallon tank </li><li>Product Tank T-102: 300-gallon tank</li><li>Feed Water Tank T-103: 20-gallon chemical tank </li></ul><p> </p><p>Del Mar College adds energy <strong>????</strong> on one side and removes it on the other. They teach their students energy in, energy out and chemical injection. At some point in the near future, the feed and product tanks will connect to the HOT3 distillation tower. <strong>????</strong> </p><p> </p><h4>HOT-3 <em>(Hands on Training Unit 3) </em>Description </h4><p> </p><p>This unit is a distillation column that distills H20 from ethylene glycol under approximately 100 mm of vacuum, with 0 being a perfect vacuum. The unit boils/distills off the water at approximately 120-125 degrees. Del Mar College teaches this unit in 5 systems or components:  bottoms, overhead, energy in, energy out and the overhead pressure system.  &nbsp;Both the HOT-1 and HOT-3 units are presently stand-alone, with plans to join them in the future. Currently the feed and product tanks are located in the tower itself which lends itself to some confusion with our students (needed).<strong>????</strong>  </p><p> </p><p><strong><em>The assets HOT-1 and HOT-3 are combinations</em></strong> of different components. The tables below provide a summary of those components and a short description. In requesting interpolated data from this dataset, it is possible to get all data for an asset or only get a subset related to a component.   </p><p> </p><h4>Hands-On-Training Unit #1 (HOT-1) </h4><p> </p><p>| Component | Description | Detail |  &nbsp;|:---------:|:-----------:|:-------| &nbsp;| <strong>E-101</strong> | <strong>Air cooler</strong>  | Cools the product water from E-102 &nbsp;| <strong>E-102</strong> | <strong>Shell and tube heat exchanger</strong> | Heats the unit feed water before entering product tank T-102 |  &nbsp;| <strong>HTR-101</strong> | <strong>Heater</strong> |  Heats product water from T-102 |  &nbsp;| <strong>TK-101</strong> | <strong>Cold side storage tank</strong> | Capacity of 300-gallons |  &nbsp;| <strong>TK-102</strong> | <strong>Hot side product tank</strong> | Capacity of 300-gallons  |  &nbsp;| <strong>TK-103</strong> | <strong>Feed water tank</strong> | Capacity of 20-gallons |  </p><h4><br></h4><h4>Hands-On-Training Unit #1 (HOT-3) </h4><p> </p><p><strong>[TO COMPLETE]</strong> &nbsp;</p><p> </p><p>| Component | Description |  </p><p>|:---------:|:-----------:| </p><p>| <strong>Bottom Products</strong> | <strong>(no description)</strong>  | </p><p>| <strong>Energy In</strong> | <strong>Reboiler system for HOT-3</strong> | </p><p>| <strong>Energy Out</strong> | <strong>Condenser System for HOT-3</strong> | </p><p>| <strong>Overhead Product</strong> | <strong>(no description)</strong> | </p><p>| <strong>Pressure</strong> | <strong>(no description)</strong> | </p><h3>| <strong>T-200</strong> | <strong>Distillation Column</strong> | </h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done